### PR TITLE
New Apps!

### PR DESCRIPTION
**Pull request by larsenv**

### CODE REVIEW

File: contents/LoadPriiloader.oscmeta [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "author": "DacoTaco",
+        "category": "utilities",
+        "peripherals": ["Wii Remote"],
+        "version": "auto"
+    },
+    "source": {
+        "type": "github_release",
+        "format": "zip",
+        "repository": "DacoTaco/priiloader",
+        "file": "LoadPriiloader.zip"
```
Comment (dhtdht020) [inline contents/LoadPriiloader.oscmeta:13]: github_release only pulls from latest stable github release, so this does not work.

File: contents/WiiGPTii.oscmeta [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                "apps/WiiGPTii/meta.xml"
+            ]
+        },
+        {
+            "treatment": "web.download",
+            "arguments": [
+                "https://github.com/dakotath/wiigptii/raw/master/Homebrew/apps/WiiGPTii/icon.png?raw=true",
+                "apps/WiiGPTii/icon.png"
+            ]
+        },
```
Comment (dhtdht020) [inline contents/WiiGPTii.oscmeta:36]:
```suggestion
        }
```
Invalid JSON syntax

File: contents/YABDM.oscmeta [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "information": {
+        "name": "Yet Another BlueDump MOD",
+        "author": "nicksasa, DarkMatterCore",
+        "category": "utilities",
+        "peripherals": ["Wii Remote", "Wii Remote", "Wii Remote", "Wii Remote", "GameCube Controller", "Classic Controller" "Nunchuk", "SDHC"],
```
Comment (dhtdht020) [inline contents/YABDM.oscmeta:6]:
```suggestion
        "peripherals": ["Wii Remote", "Wii Remote", "Wii Remote", "Wii Remote", "GameCube Controller", "Classic Controller", "Nunchuk", "SDHC"],
```
Invalid JSON syntax

File: contents/WiiLauncher.oscmeta [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        "author": "conanac",
+        "category": "utilities",
+        "peripherals": ["Wii Remote", "SDHC"],
+        "version": "0.2"
+    },
+    "source": {
+        "type": "url",
+        "format": "zip",
+        "location": "https://wiibrew.org/w/images/1/12/Wiilauncher_r11.zip"
+    }
```
Comment (dhtdht020) [inline contents/WiiLauncher.oscmeta:13]:
```suggestion
    },
```
Invalid JSON syntax

File: contents/gcmm.oscmeta [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        "author": "suloku",
+        "category": "utilities",
+        "peripherals": ["Wii Remote", "GameCube Controller", "SDHC"],
+        "version": "auto"
+    },
+    "source": {
+        "type": "github_release",
+        "format": "zip",
+        "repository": "suloku/gcmm",
+        "file": "gcmm_v*.zip"
```
Comment (dhtdht020) [inline contents/gcmm.oscmeta:13]:
```suggestion
        "file": "gcmm_*.zip"
```
There is no "v" in the release file name

**Review by dhtdht020 — CHANGES_REQUESTED** [review]

File: contents/LoadPriiloader.oscmeta (hunk 1 above)
Comment (larsenv) [reply contents/LoadPriiloader.oscmeta:13]: no u

File: contents/WiiGPTii.oscmeta (hunk 2 above)
Comment (larsenv) [reply contents/WiiGPTii.oscmeta:36]: no u

File: contents/WiiLauncher.oscmeta (hunk 4 above)
Comment (larsenv) [reply contents/WiiLauncher.oscmeta:13]: no u

File: contents/gcmm.oscmeta (hunk 5 above)
Comment (larsenv) [reply contents/gcmm.oscmeta:13]: no u

File: contents/yabause.oscmeta
Comment (dhtdht020) [inline contents/yabause.oscmeta:1]: treatments should be used here to organize the zip

**Comment by dhtdht020** [comment]
very mature replies straight to my email. so are you going to fix the issues or should I close this?

**Comment by dhtdht020** [comment]
looks good to me, merging.

**Review by dhtdht020 — APPROVED** [review]

**Pull request merged**